common/images/graphql/braintree-vault.svg

```diff
@@ -0,0 +1,3 @@
: nowrap; overflow-wrap: normal; text-align: left;"><div xmlns="http://www.w3.org/1999/xhtml" style="display:inline-block;text-align:inherit;text-decoration:inherit;white-space:normal;"><p style="font-size: 13px">Braintree sends the response to Magento.</p></div></div></foreignObject><text x="72" y="34" fill="#000000" text-anchor="middle" font-size="13px" font-family="Helvetica">[Not supported by viewer]</text></switch></g><ellipse cx="418" cy="432" rx="12" ry="12" fill="#bd3c39" stroke="none" pointer-events="none"/><g transform="translate(414.5,423.5)"><switch><foreignObject style="overflow:visible;" pointer-events="all" width="7" height="14" requiredFeatures="http://www.w3.org/TR/SVG11/feature#Extensibility"><div xmlns="http://www.w3.org/1999/xhtml" style="display: inline-block; font-size: 14px; font-family: Helvetica; color: rgb(255, 255, 255); line-height: 1.2; vertical-align: top; width: 9px; white-space: nowrap; overflow-wrap: normal; text-align: center;"><div xmlns="http://www.w3.org/1999/xhtml" style="display:inline-block;text-align:inherit;text-decoration:inherit;white-space:normal;"><div style="text-align: center ; font-size: 14px">6</div></div></div></foreignObject><text x="4" y="14" fill="#FFFFFF" text-anchor="middle" font-size="14px" font-family="Helvetica">[Not supported by viewer]</text></switch></g><g transform="translate(438.5,404.5)"><switch><foreignObject style="overflow:visible;" pointer-events="all" width="149" height="55" requiredFeatures="http://www.w3.org/TR/SVG11/feature#Extensibility"><div xmlns="http://www.w3.org/1999/xhtml" style="display: inline-block; font-size: 13px; font-family: Helvetica; color: rgb(0, 0, 0); line-height: 1.2; vertical-align: top; width: 149px; white-space: nowrap; overflow-wrap: normal; text-align: left;"><div xmlns="http://www.w3.org/1999/xhtml" style="display:inline-block;text-align:inherit;text-decoration:inherit;white-space:normal;"><p style="font-size: 13px">Magento sends an authorization request.</p></div></div></foreignObject><text x="75" y="34" fill="#000000" text-anchor="middle" font-size="13px" font-family="Helvetica">[Not supported by viewer]</text></switch></g><path d="M 378 464 L 521 464 L 595.53 464" fill="none" stroke="#cc0000" stroke-width="2" stroke-miterlimit="10" stroke-dasharray="2 2" pointer-events="none"/><path d="M 589.76 468 L 597.76 464 L 589.76 460" fill="none" stroke="#cc0000" stroke-width="2" stroke-miterlimit="10" pointer-events="none"/><path d="M 381.97 484 Q 381.97 484 597.5 484" fill="none" stroke="#cc0000" stroke-width="2" stroke-miterlimit="10" stroke-dasharray="2 2" pointer-events="none"/><path d="M 387.74 480 L 379.74 484 L 387.74 488" fill="none" stroke="#cc0000" stroke-width="2" stroke-miterlimit="10" pointer-events="none"/><path d="M 51 570 L 51 570" fill="none" stroke="#000000" stroke-miterlimit="10" pointer-events="none"/><path d="M 51 570 L 51 570 L 51 570 L 51 570 Z" fill="#000000" stroke="#000000" stroke-miterlimit="10" pointer-events="none"/></g></svg>
```

Comment:
For (3) The client will provide a `public_hash` from the `customerPaymentTokens` instead of a nonce.